packages/telemetry/src/context-aware-slog.js

```diff
@@ -178,7 +165,15 @@ export const logCreator = (emitLog, persistenceUtils = {}) => {
         };
         break;
       }
-      case SLOG_TYPES.COSMIC_SWINGSET.BOOTSTRAP_BLOCK.FINISH: {
+      case SLOG_TYPES.COSMIC_SWINGSET.END_BLOCK.START: {
+        assert(!!blockContext);
+        break;
+      }
```

Comment:
There should be no trigger context here either (since we're restoring it later)

```suggestion
      case SLOG_TYPES.COSMIC_SWINGSET.END_BLOCK.START:
```